lib/u3d/unity_versions.rb

```diff
@@ -148,10 +150,30 @@ def list_available
           end
           data.gsub(/[ \t]+/, '').each_line { |l| puts l if /<a href=/ =~ l }
           versions = {}
-          results = data.scan(LINUX_DOWNLOAD)
+          results = data.scan(LINUX_DOWNLOAD_DATED)
           results.each do |capt|
             versions[capt[1]] = capt[0]
           end
+
+          results = data.scan(LINUX_DOWNLOAD_RECENT_PAGE)
+          results.each do |page|
+            url = page[0]
+            uri = URI(url)
+            Net::HTTP.start(uri.host, uri.port) do |http|
+              request = Net::HTTP::Get.new uri
+              response = http.request request
+            end
+            if response.kind_of? Net::HTTPSuccess
+              capt = response.body.match(LINUX_DOWNLOAD_RECENT_FILE)
+              if capt && capt[1] && capt[2]
+                versions[capt[2].gsub(/x/, '')] = capt[1]
+              else
+                UI.error("Could not retrieve a fitting file from #{url}")
```

Comment:
I'll accept the patch as is. For future improvements, I wonder if we should fail harder here (throw exceptions)